Tests/AppTests/WebpageSnapshotTestCase.swift

```diff
@@ -23,6 +24,8 @@ class WebpageSnapshotTestCase: SnapshotTestCase {
     override func setUpWithError() throws {
         try super.setUpWithError()
 
+        SnapshotTesting.diffTool = "ksdiff"
```

Comment:
Just a little convenience to make Xcode print a copy-pastable `ksdiff` command in the console on snapshot failure.